src/compiler/transformers/ts.ts

```diff
@@ -1210,53 +1248,21 @@ namespace ts {
          * @param receiver The object receiving the property assignment.
          */
         function transformInitializedProperty(property: PropertyDeclaration, receiver: LeftHandSideExpression) {
-            const propertyName = visitPropertyNameOfClassElement(property);

```

Comment:
We don't need to do this for fields anymore?

Reply:
Correct. The work is done inside the hoisting of computed names/decorated names. If you still do it, you get redundant output like so:
```ts
        this[_a = _c] = null;
        this[_b = _f] = null;
        var _a, _b;
```
So we know we don't need to transform computed property names on initialized properties because we've already dealt with them.